splashsurf/src/io.rs

```diff
@@ -167,3 +160,22 @@ pub fn to_binary_f32<R: Real, P: AsRef<Path>>(file: P, values: &[R]) -> Result<(
 
     Ok(())
 }
+
+// #[repr(transparent)]
+struct Vec3r(Vector3<f32>);
```

Comment:
I think we should rename this to Vec3f32 for now, because currently we only support reading PLY files with floats.

Reply:
+ you can remove the repr attribtute